StripeUICore/StripeUICore/Source/Helpers/StripeUICoreBundleLocator.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 
 @_spi(STP) public final class StripeUICoreBundleLocator: BundleLocatorProtocol {
     public static let internalClass: AnyClass = StripeUICoreBundleLocator.self
-    public static let bundleName = "StripeUICore"
+    public static let bundleName = "StripeUICoreBundle"
```

Comment:
We had some install tests failing: https://github.com/CocoaPods/CocoaPods/issues/12051

Reply:
Good idea, added!